lib/plugins/aws/customResources/resources/eventBridge/lib/utils.js

```diff
@@ -2,7 +2,8 @@
 
 function getEventBusName(eventBus) {
   if (eventBus && eventBus.startsWith('arn')) {
-    return eventBus.split('/').pop();
+    const separatorIndex = eventBus.indexOf('/');
+    return separatorIndex === -1 ? eventBus : eventBus.slice(separatorIndex + 1);
```

Comment:
Looks good now.

Still, is this possible for event bus _arn_ to not contain `/` character?

In tests I don't see such case explored.

I'm thinking that maybe we should have here `eventBus.slice(eventBus.indexOf('/')+ 1)` unconditionally(?)

Reply:
Every `arn` has at least one `/` which is basically the separator for the name of the the event bus.

Reply:
Updating...